examples/gno.land/p/demo/users/users.gno

```diff
@@ -0,0 +1,31 @@
+package users
+
+import (
+	"std"
+	"strconv"
+)
+
+//----------------------------------------
+// Types
+
+type User struct {
+	Address std.Address
+	Name    string
+	Profile string
```

Comment:
Are fields like profile, number, invites, and inviter needed as part of a basic `User` definition?  These seem to be application specific.  Or at the very least I should be able to write an application that can couple a user's address with its username without the extra fields I don't care about; that seems to be the base functionality.

But it also might be helpful to provide a struct like `UserData` or `Extended` user that provides the extra fields for like applications that want to operate using this data model:
```
type ExtendedUser struct {
    User
    Profile string
    Number int
    Invites int
    Inviter std.Address
}
```

Reply:
@deelawn agreed, I think it makes sense to keep those fields relevant to r/demo/users (when rendering pages and using its inviting system) so that third party realms/packages only get what they need most of the time (a name associated to an address)